lib/ruby_lsp/store.rb

```diff
@@ -22,25 +22,23 @@ def initialize
 
     sig { params(uri: URI::Generic).returns(Document) }
     def get(uri)
-      path = uri.to_standardized_path
-      return T.must(@state[T.must(uri.opaque)]) unless path
-
-      document = @state[path]
+      document = @state[uri.to_s]
```

Comment:
Just to save a little repetition in calling `to_s` each time, not a big deal.

Reply:
I tried it, but it ends up not producing the expected results. If a URI is parsed using `URI("path")`, it will produce a `URI::File` object. If we build it using `URI::Generic.build`, it produces a `URI::Generic` object.

In that scenario, even if both URIs have the exact same address and scheme, they will produce different values for `hash` and will not land on the same key. I think we're safer using `to_s`.